.github/workflows/auto-triage-issues

```diff
@@ -0,0 +1,20 @@
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  auto_label:
+    runs-on: ubuntu-latest
+    name: Automatic Github Issue Labeller
+    steps:
+    - name: Label Step
+      uses: larrylawl/Auto-Github-Issue-Labeller@main
+      with:
+        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        REPOSITORY: ${{github.repository}}
+        DELTA: "1"
+        CONFIDENCE: "2"
+        FEATURE: "enhancement"
+        BUG: "bug"
+        DOCS: "documentation"
+        VERSION: "v1.0"
```

Comment:
What's the significance of this version field?

Reply:
Two version of the action are available. This one gets chosen by default

Reply:
So it's not something that we should specify as a parameter right?

Reply:
Yeah it's optional. We can leave it if we want

Reply:
Removed it.